samples/subsys/llext/modules/README.rst

```diff
@@ -0,0 +1,59 @@
+.. zephyr:code-sample:: llext-modules
+   :name: Linkable loadable extensions "module" sample
+   :relevant-api: llext
+
+    Call a function in a loadable extension module,
+    either built in or loaded at runtime.
+
+Overview
+********
+
+This sample demonstrates the use of the :ref:`llext` subsystem in Zephyr. The
+llext subsystem allows for the loading of relocatable ELF files at runtime;
+their symbols can be accessed and functions called.
+
+Specifically, this shows how to call a simple "hello world" function,
+implemented in ``src/hello_world_ext.c``. This is achieved in two different ways,
```

Comment:
nit (i.e. not worth updating unless there's another round of reviews needed): maybe make this a `:zephyr_file:` like you're doing for others